src/shared/components/ResourceEditor/useEditorTooltip.tsx

```diff
@@ -223,7 +219,7 @@ function useEditorTooltip({
           tooltip.remove();
         }
         return clearTimeout(timeoutId);
-      }, 3000);
+      }, 2000);
```

Comment:
If the timeout is for what I think it is (I think it is to hide the tooltip that is shown when user hovers on link), then I think 3 seconds might be a better timeout than 2 because it would give more time to the user to read the contents of the tooltip.

Reply:
yeah completely agree, I was just doing an experiment and forget about it